images/orbit-controller/src/orbit_controller/operators/orbitjob_operator.py

```diff
@@ -167,3 +167,54 @@ def create_job(
         "orbitJobOperator": {"jobStatus": "JobCreated", "jobName": job_instance_metadata.name, "nodeType": node_type}
     }
     return "JobCreated"
+
+
+@kopf.on.timer(
+    ORBIT_API_GROUP,
+    ORBIT_API_VERSION,
+    "orbitjobs",
+    interval=5,
+    initial_delay=20
+    # when=_should_index_jobs
```

Comment:
we need a filter to stop the timers when a Job is completed, either successfully or failed. without a filter the timer will continue to fire for each OrbitJob every 5 seconds

Reply:
a good filter might be to check the jobs_idx, if no Job exists belonging to the OrbitJob, then don't fire